pages/pipelines/tags.md

```diff
@@ -0,0 +1,13 @@
+# Pipeline tags
+
+Pipeline tags allow you to tag and search for your pipelines using the search bar. Tags are beneficial when you have many pipelines and would like to group and filter through them quickly.
+
+<%= image "pipeline-tag-search.png", alt: "Imagine showing pipeline tag search" %>
+
+## Using tags
+
+Tags are unique per pipeline. You can only add the same tag once. Tags are limited to five per pipeline, and are no longer than 64 chars. Tag names support emoji and text.
```

Comment:
Just some general suggestions that I think improve the flow. I've taken the chance to add a recommendation so check that it makes sense, please haha.

```suggestion
You can assign each pipeline up to five unique tags. A tag can comprise emoji and text, up to 64 characters. We recommend using an emoji to make the tag stand out and keeping the text short and clear.
```